cmd/contour/shutdownmanager.go

```diff
@@ -290,6 +309,7 @@ func registerShutdown(cmd *kingpin.CmdClause, log logrus.FieldLogger) (*kingpin.
 
 	shutdown := cmd.Command("shutdown", "Initiate an shutdown sequence which configures Envoy to begin draining connections.")
 	shutdown.Flag("admin-port", "Envoy admin interface port.").IntVar(&ctx.adminPort)
```

Comment:
seems like this flag should be noted as deprecated?

Reply:
marked as deprecated!